src/include/btmem.h

```diff
@@ -555,6 +555,21 @@ struct __wt_col_fix_tw {
 /* WT_COL_FIX_TW_CELL gets the cell pointer from a WT_COL_FIX_TW_ENTRY. */
 #define WT_COL_FIX_TW_CELL(page, entry) ((WT_CELL *)((uint8_t *)(page)->dsk + (entry)->cell_offset))
 
+#ifdef HAVE_DIAGNOSTIC
+/*
+ * WT_SPLIT_HIST --
+ *	State information of a split at a single point in time.
+ */
+struct __wt_split_page_hist {
+    const char *name;
+    const char *func;
+    uint64_t split_gen;
+    uint32_t entries;
```

Comment:
Is there a way to identify the page that is split? Can we record the pointer to the ref?

Reply:
The split is saved in the page itself.